src/form/wr.jl

```diff
@@ -337,7 +337,7 @@ end
 
 
 "do nothing by default but some formulations require this"
-function variable_current_storage(pm::AbstractWRModel; nw::Int=pm.cnw, bounded::Bool=true, report::Bool=true)
+function variable_current_storage(pm::AbstractWModels; nw::Int=pm.cnw, bounded::Bool=true, report::Bool=true)
```

Comment:
Same as above, `AbstractWConvexModels`.  Also now as a shared variable definition, it can move to `form/shared.jl`.